scimono-server/src/test/java/com/sap/scimono/entity/bulk/BulkResponseOperationLocationServiceTest.java

```diff
@@ -0,0 +1,32 @@
+package com.sap.scimono.entity.bulk;
+
+import com.sap.scimono.helper.ResourceLocationService;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.Arrays;
+
+@RunWith(MockitoJUnitRunner.class)
+public class BulkResponseOperationLocationServiceTest {
+
+  private ResourceLocationService usersLocationService;
+  private ResourceLocationService groupsLocationService;
+
+  @DisplayName("Test Bulk request with multiple POST methods with null bulkId returns invalid operation response")
+  @Test
+  public void validateBulkIdIsRequiredForPostMethodTest() {
+    ResponseOperation responseOperation = buildResponseOperation(null);
+    BulkBody<ResponseOperation> bulkResponse = BulkBody.forResponse(Arrays.asList(responseOperation, responseOperation));
+    BulkResponseOperationLocationService responseService = new BulkResponseOperationLocationService(usersLocationService, groupsLocationService);
+    BulkBody<ResponseOperation> response = responseService.rebuildWithLocations(bulkResponse);
+    Assert.assertEquals("The number of operations is not as same as expected.", 2, response.getOperations().size());

```

Comment:
Could you also explicitly assert that the ResponseOperations have no locations so that anybody reading the test understands the intent of the test? Currently this is done implicitly by supplying null values for usersLocationService and groupsLocationService.